collectors/python.d.plugin/spigotmc/spigotmc.chart.py

```diff
@@ -52,7 +52,7 @@
     re.X
 )
 _LIST_REGEX = re.compile(
-    r'(\d+)',  # Current user count.
+    r'[^§](\d+)(?:.*?(?=/).*?[^§](\d+))?',  # Current user count.
```

Comment:
> §6当前有 §c2§6 个玩家在线,最大在线人数为 §c50§6 个玩家.

please add some examples there as comments

Reply:
When I execute list on the server, I see this output:
![Jietu20190811-173633](https://user-images.githubusercontent.com/9296576/62832299-a607fd80-bc5e-11e9-9c92-b775256c8b2a.png)
But the actual output obtained by rcon is actually like this:
```
§6当前有 §c5§6 个玩家在线,最大在线人数为 §c50§6 个玩家.
```
For example, `§c` for red and `§6` for yellow

If you change the previous one, you will mistake the result of 6

Links:
* [Analysis of the new regular](https://jex.im/regulex/#!flags=&re=%5B%5E§%5D(%5Cd%2B)(%3F%3A.*%3F(%3F%3D%2F).*%3F%5B%5E§%5D(%5Cd%2B))%3F)
* [New regular test](https://regex101.com/r/y3zvGG/4)
    * Look at the first result of each line below (Match 1, 3, 5, 7, 9, 11)

Reply:
Oh, I remember, the color should be the reason for Essentials, but this plugin is very common, and the new code does not break the compatibility of the native.

Reply:
I found an easier way to solve the list problem.

Reply:
You can also try the native output, similar to the following line

```
There are 4 of a max 50 players online: woozy, Absurd_7, Sheep, BSoD_Ultimate
```

It works fine

Reply:
My suggestion is to add test strings as comments right before the regex. Thx for the links, good stuff :+1: